.mocharc.json

```diff
@@ -0,0 +1,3 @@
+{
+    "bail": true
```

Comment:
is this change really needed? it looks like `true` is default value for bail (see https://mochajs.org/api/mocha#bail)

Reply:
No, it is not.